src/alert.sh

```diff
@@ -0,0 +1,22 @@
+# NOTE: src/commons.sh must be included before this file
+
+# Alerts command completion to the user. First argument is the alert code,
+# and second is the command string to be printed in visual notification.
+function alert_completion()
+{
+
+  local COMMAND=$1
+  local ALERT_OPT=$2
+
+  if [[ $# -gt 1 && "$2" =~ ^--alert= ]]; then
+    local opts="$(echo $2 | sed s/--alert=// )"
+
+    grep -o . <<< "$opts" | while read option;  do
+      if [ "$option" == "v" ]; then
+        eval "${configurations[visual_alert_command]}"
+      elif [ "$option" == "s" ]; then
+        eval "${configurations[alert_player]}" "${configurations[alert_sound_path]}" &
```

Comment:
Add the "&" to the "v" option as well

Reply:
Nice, I did know about that because I'm not using notify-send. Anyway, did you test it?

Reply:
Yes, it's tested :) (but we don't have automatic tests for this feature yet)